source/features/action-used-by-link.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as pageDetect from '../libs/page-detect';
+import searchIcon from 'octicon/search.svg';
```

Comment:
```suggestion
import searchIcon from 'octicon/search.svg';
import * as pageDetect from '../libs/page-detect';
```

Tip 👍 https://github.com/sindresorhus/refined-github/pull/2881#discussion_r392447778
> Sort these so they look like a tree 😃 refer to the look in my other features. This makes it slightly easier to follow because they’re closer to being “aligned”
> 
> And always put the npm modules above the local `./` and `../` files

Reply:
This is great! :evergreen_tree:

Reply:
> And always put the npm modules above the local `./` and `../` files

Reply:
oops, sorry, I should have paid more attention.